geomstats/geometry/full_rank_correlation_matrices.py

```diff
@@ -337,3 +350,187 @@ def __init__(self, space):
         diffeo = ComposedDiffeo(diffeos)
 
         super().__init__(space=space, diffeo=diffeo, image_space=image_space)
+
+
+def off_map(matrix):
+    """Subtract diagonal to a matrix."""
+    return matrix - Matrices.to_diagonal(matrix)
+
+
+class OffLogDiffeo(Diffeo):
+    """Off-log diffeomorphism from Cor+ to Hol.
+
+    A diffeomorphism between full-rank correlation matrices Cor+ and
+    symmetric hollow matrices Hol.
```

Comment:
Add a reference, or an equation, or an explanation of what off-log is.

Parameter description missing.